features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/utils/ImageWireframeHelper.kt

```diff
@@ -65,6 +65,8 @@ interface ImageWireframeHelper {
      * @param shapeStyle provides a custom shape (e.g. rounded corners) to the image wireframe
      * @param border provides a custom border to the image wireframe
      * @param prefix a prefix identifying the drawable in the parent view's context
+     * @param customResourceIdCacheKey an optional key with which to cache or retrieve from the resource cache.
+     * If this key is not provided then one will be generated from the drawable.
      */
     // TODO RUM-3666 limit the number of params to this function
     fun createImageWireframeByDrawable(
```

Comment:
side note: it is a bit strange that `detekt` doesn't flag this method as a method with long list of arguments. Do we have this rule enabled?